tests/manager/manager-executeQuery_error-002.phpt

```diff
@@ -1,7 +1,5 @@
 --TEST--
 MongoDB\Driver\Manager::executeQuery() with invalid options
---SKIPIF--
-<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
```

Comment:
I think we also want `skip_if_not_live()` here. Although [read preference validation](https://github.com/mongodb/mongo-php-driver/blob/1.6.0alpha3/src/MongoDB/Manager.c#L474) occurs _before_ server selection (and for some reason [again in `phongo_execute_query()`](https://github.com/mongodb/mongo-php-driver/blob/1.6.0alpha3/php_phongo.c#L852)), [session validation](https://github.com/mongodb/mongo-php-driver/blob/1.6.0alpha3/php_phongo.c#L859) only occurs _after_ server selection.

I'm not sure how this test could pass without a live server.

Reply:
With the introduction of mongos pinning, the session is also validated [before server selection](https://github.com/mongodb/mongo-php-driver/blob/a85e0922c9d1c2a82fb02eef6625d75b02721a79/src/MongoDB/Manager.c#L512).

The reason the equivalent test for the server needs the `skip_if_not_live` check is that we perform server selection manually in the test.